index.js

```diff
@@ -8,9 +8,19 @@ var unwatchedTree = require('broccoli-unwatched-tree');
 module.exports = {
   name: 'ember-cli-mirage',
 
-  included: function included(app) {
-    this._super.included.apply(this, arguments);
+  options: {
+    nodeAssets: {
```

Comment:
@josemarluedke It's handled by https://emberobserver.com/addons/ember-cli-node-assets, which basically just provides some sugar for funneling files from `node_modules` into your build to either be `app.import`ed or land in `public`.